src/test/java/org/javarosa/entities/EntitiesTest.java

```diff
@@ -313,35 +311,4 @@ public void whenSaveToQuestionIsNotAnswered_entityPropertyIsEmptyString() throws
         assertThat(entities.size(), equalTo(1));
         assertThat(entities.get(0).properties, equalTo(asList(new Pair<>("name", ""))));
     }
-
-    @Test
-    public void savetoIsRemovedFromBindAttributesForClients() throws IOException, XFormParser.ParseException {
```

Comment:
I don't know why this was important? Happy to be corrected, but I don't see any reason to not allow accessing `saveto` from `DataBinding`.

Reply:
Looks like it was introduced at the very beginning in https://github.com/getodk/javarosa/pull/691. I don't see a specific explanation but in my head it makes sense to want confidence that entity directives are handled by the JR processor and not available for clients to muck with. I see it as a nice way to enforce hygiene which is something that Collect has had trouble with in the past.

Reply:
That's fair. I'm not feeling as concerned about that now. Introducing these processors has given us a structured way to interact with form parsing, and my gut reaction is that removing attributes feels overly protective.

Reply:
Sounds reasonable to me!